crates/forge/src/tests_to_run.rs

```diff
@@ -0,0 +1,29 @@
+use test_collector::TestCase;
+
+#[derive(Debug, PartialEq)]
+pub(crate) enum TestsToRun {
+    All,
+    Ignored,
+    NotIgnored,
+}
```

Comment:
```suggestion
#[derive(Debug, PartialEq)]
// Specifies what tests should be excluded based on `#[ignore]` attribute
pub(crate) enum  ExcludeIgnore {
    None,
    Ignored,
    NotIgnored,
}
```
Would be more consistent with other filtering mechanisms